dgl_sparse/src/sparse_matrix.cc

```diff
@@ -96,65 +96,89 @@ std::shared_ptr<CSR> SparseMatrix::CSCPtr() {
 std::vector<torch::Tensor> SparseMatrix::COOTensors() {
   auto coo = COOPtr();
   auto val = value();
+  if (coo->data.has_value()) {
+    val = val.index({coo->data.value()});
+  }
```

Comment:
I still don't think this is the intended behavior. `COOTensors()` just need to return row and col. `CSRTensors()` returns indptr, indices and value_indices. On the user side, if they want value, they just get it by `A.val`, so it is kinda redundant to return the value tensors here.

Reply:
These interfaces return `value` here. Do you mean it is better to return `value_indices`?